index.php

```diff
@@ -0,0 +1,49 @@
+<?php
+session_start();
```

Comment:
why are we starting session before we have validated if user has logged in or not?

Reply:
To access session variable I used session_start(). I also used session_status(), but didn't worked.